src/gui/optionsdialog.ui

```diff
@@ -779,6 +779,27 @@
               </layout>
              </widget>
             </item>
+            <item>
+             <widget class="QCheckBox" name="checkPreallocateAll">
+              <property name="text">
+               <string>Pre-allocate disk space for all files</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QCheckBox" name="checkAppendqB">
+              <property name="text">
+               <string>Append .!qB extension to incomplete files</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QCheckBox" name="checkRecursiveDownload">
+              <property name="text">
+               <string>Enable recursive download dialog</string>
```

Comment:
Maybe just "Enable recursive download"?

Reply:
I don't think it is better, this option doesn't really do recursive download, it just shows a dialog instead.

Reply:
But the dialog is the only way currently to use "recursive download" feature, isn't it? So enabling/disabling the dialog enables/disables the entire feature.

Reply:
>Also it might give users the wrong idea recursive download will be automatic, while it is not.

It's not clear for me what is this comment applied to. From my POV, it's about current wording. When I uncheck "Enable recursive download dialog" I assume I disable the dialog only but not the feature itself.

Reply:
Or maybe "Ask for recursive download"?

Reply:
It's also unclear in case of "don't ask"... Don't ask and what? Do it or not? It sounds like "Recursive download it without asking". Maybe I'm taking this the wrong way? What can most users think?

Reply:
@thalieht
Any comment?

Reply:
In short, i would keep the dialog.
I think both of you are right. I can't speak for other users but without the "dialog" i would think that it's automatic (actually i would be wondering what that option is until i first encounter such a torrent). And with the "dialog" well, i would await for a dialog to appear to find out :)

Reply:
Well, since the rest is OK for me, I'll approve it for the case you decide to leave it as it is.

Reply:
Or maybe just extract this checkbox into separate PR?

Reply:
>Or maybe just extract this checkbox into separate PR?

IMO we can have it first in codebase and change it later if some better idea comes up.